### ISSUE TITLE
Allow underscores and hyphens in class names

### ISSUE THREAD

**Issue by GitBruno** [mwilliamson/python-mammoth#24]
At the moment these are not supported in style maps.

It would be nice if a custom style map does not return an error for valid chars.

```
    Did not understand this style mapping, so ignored it:
    p[style-name='15 RomText Open'] => p.15-RomText-Open

```

**Comment by mwilliamson** [comment]
The rules are the same as in CSS: identifiers cannot start with a digit. If the class name starts with a digit, use a backslash to escape them e.g.

```
p[style-name='15 RomText Open'] => p.\15-RomText-Open
```

**Comment by mwilliamson** [comment]
Whoops, slight correction: being able to escape identifiers was in the development version, which has just been released as 1.3.0.

**Comment by GitBruno** [comment]
Cool! Ill check it out
